<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="2" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Capturing Screenshot]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href="D:\Screenshot\error.png">Screenshot</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href="D:\Screenshot\error.png"><img src="D:\Screenshot\error.png" height=200 width=200></img></a>]]>
    </line>
    <line>
      <![CDATA[Add Customer Successfully]]>
    </line>
    <line>
      <![CDATA[Capturing Screenshot]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href="D:\Screenshot\error.png">Screenshot</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href="D:\Screenshot\error.png"><img src="D:\Screenshot\error.png" height=200 width=200></img></a>]]>
    </line>
  </reporter-output>
  <suite name="Data Driven Framework Practice" duration-ms="9753" started-at="2018-08-21T02:40:31Z" finished-at="2018-08-21T02:40:41Z">
    <groups>
    </groups>
    <test name="Bank Manager LoginTest" duration-ms="879" started-at="2018-08-21T02:40:31Z" finished-at="2018-08-21T02:40:32Z">
      <class name="com.infofactors.testcaes.BankManagerLoginTest">
        <test-method status="FAIL" signature="loginAsBankManager()[pri:0, instance:com.infofactors.testcaes.BankManagerLoginTest@1613674b]" name="loginAsBankManager" duration-ms="785" started-at="2018-08-21T02:40:31Z" finished-at="2018-08-21T02:40:32Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Bank Manger Login Fail]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Bank Manger Login Fail
at org.testng.Assert.fail(Assert.java:96)
at com.infofactors.testcaes.BankManagerLoginTest.loginAsBankManager(BankManagerLoginTest.java:30)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Capturing Screenshot]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href="D:\Screenshot\error.png">Screenshot</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href="D:\Screenshot\error.png"><img src="D:\Screenshot\error.png" height=200 width=200></img></a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginAsBankManager -->
      </class> <!-- com.infofactors.testcaes.BankManagerLoginTest -->
    </test> <!-- Bank Manager LoginTest -->
    <test name="Add Customer Test" duration-ms="8587" started-at="2018-08-21T02:40:32Z" finished-at="2018-08-21T02:40:41Z">
      <class name="com.infofactors.testcaes.AddCustomerTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.infofactors.testcaes.AddCustomerTest@3fb1549b]" name="setUp" is-config="true" duration-ms="19032" started-at="2018-08-21T02:40:12Z" finished-at="2018-08-21T02:40:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="addCustomerTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.infofactors.testcaes.AddCustomerTest@3fb1549b]" name="addCustomerTest" duration-ms="8438" started-at="2018-08-21T02:40:32Z" data-provider="getData" finished-at="2018-08-21T02:40:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[praveen]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[shaik]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[123456.0]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Customer added successfully]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Add Customer Fail]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Add Customer Fail
at org.testng.Assert.fail(Assert.java:96)
at com.infofactors.testcaes.AddCustomerTest.addCustomerTest(AddCustomerTest.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Add Customer Successfully]]>
            </line>
            <line>
              <![CDATA[Capturing Screenshot]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href="D:\Screenshot\error.png">Screenshot</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href="D:\Screenshot\error.png"><img src="D:\Screenshot\error.png" height=200 width=200></img></a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- addCustomerTest -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.infofactors.testcaes.AddCustomerTest@3fb1549b]" name="teardown" is-config="true" duration-ms="1847" started-at="2018-08-21T02:40:41Z" finished-at="2018-08-21T02:40:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.infofactors.testcaes.AddCustomerTest -->
    </test> <!-- Add Customer Test -->
  </suite> <!-- Data Driven Framework Practice -->
</testng-results>
